spec/resources/invoices/invoices_api_spec.rb

```diff
@@ -12,7 +12,15 @@
   end
 
   it "is valid with certain attributes" do
-    sales_invoice = Elmas::SalesInvoice.new(journal:"my-awesome-journal", ordered_by: "1230")
+    sales_invoice = Elmas::SalesInvoice.new(
+      journal:"my-awesome-journal",
+      ordered_by: "1230",
+      invoice_date: Time.now,
```

Comment:
I know this is just filling some fields to be present, but try to use `zone` whenever you use `Time`.

```suggestion
      invoice_date: Time.zone.now,
```

Reply:
We are not in Rails land! `zone` comes from activesupport